src/bosh-director/spec/unit/api/controllers/tasks_controller_spec.rb

```diff
@@ -399,8 +399,26 @@ module Api
               expect(last_response.headers['Content-Range']).to eq('bytes 5-10/11')
             end
 
+            it 'has API call that return task output from db with ranges' do
+              task1 = Models::Task.make(state: 'queued', description: 'fake-description', event_output: "Test output", output: "something")
+
+              # Range test
+              get "/#{task1.id}/output?type=event", {}, {'HTTP_RANGE' => 'bytes=0-3'}
+              expect(last_response.status).to eq(206)
+              expect(last_response.body).to eq('Test')
+              expect(last_response.headers['Content-Length']).to eq('4')
+              expect(last_response.headers['Content-Range']).to eq('bytes 0-3/11')
+
+              # Range test
+              get "/#{task1.id}/output?type=event", {}, {'HTTP_RANGE' => 'bytes=5-'}
+              expect(last_response.status).to eq(206)
+              expect(last_response.body).to eq('output')
+              expect(last_response.headers['Content-Length']).to eq('6')
+              expect(last_response.headers['Content-Range']).to eq('bytes 5-10/11')
+            end
+
             it 'supports returning different types of output (debug, cpi, event)' do
-              %w(debug event cpi).each do |log_type|
```

Comment:
Should the test name also mention `result`? Also the new `result output` behavior of persisting to the db is not being tested.